backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/EnrollmentController.java

```diff
@@ -44,13 +45,9 @@ public EnrollSubscriberResponse enrollSubscriber(@Valid @RequestBody EnrollSubsc
 		
 		logger.info("Subscriber enroll request: {} ", enrollSubscriberRequest.getPhn());
 		
-		//Convert request to R50 message and get it as HL7 v2 String. (This message is used to send to external endpoint wrapped in JSON)
-		R50 r50 = convertEnrollSubscriberRequestToR50(enrollSubscriberRequest);
-		
-		//Send to R50 endpoint
-		Message message = enrollmentService.enrollSubscriber(r50);
-		
-		//Convert R50 endpoint response to HN Web response
+		R50 r50 = convertEnrollSubscriberRequestToR50(enrollSubscriberRequest);		
+		final String transactionId = UUID.randomUUID().toString();		
```

Comment:
I don't tend to bother adding final to local variables as it clutters up the code. Effectively, I think it's only a hint to developers as the compiler will infer this anyways.

Reply:
Removed final